### ISSUE TITLE
SQS Message Group Id should be parsed as String

### ISSUE THREAD

**Issue by davidrusu** [ex-aws/ex_aws_sqs#24]
### Environment

* Elixir & Erlang versions (elixir --version): 
erlang 22.3.3
elixir 1.9.4-otp-22

* ExAws version `mix deps |grep ex_aws`
```
>mix deps |grep ex_aws
* ex_aws (Hex package) (mix)
  locked at 2.1.3 (ex_aws) 0bdbe2ae
* ex_aws_dynamo (Hex package) (mix)
  locked at 3.0.2 (ex_aws_dynamo) 7233b402
* ex_aws_sqs (Hex package) (mix)
  locked at 3.2.1 (ex_aws_sqs) ae77e296
```

* HTTP client version. IE for hackney do `mix deps | grep hackney`
```
* hackney (Hex package) (rebar3)
  locked at 1.16.0 (hackney) 3bf0bebb
```

### Current behavior

ex_aws_sqs will currently parse an all numerical group id (ie. "111111111") as an integer. The SQS docs do say that a message group id should be a String: https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_SendMessage.html

We have code that would downcase the group id while processing messages: (here the group id for the message was "00001111111111111111", note that the first three `0`'s have been cut-off.
```
GenServer #PID<0.23161.15> terminating
** (FunctionClauseError) no function clause matching in String.downcase/2
    (elixir) lib/string.ex:744: String.downcase(1111111111111111, :default)
```

### Expected behavior

MessageGroupId's should always be parsed as Strings

**Comment by TattdCodeMonkey** [comment]
This is likely caused by `try_cast/1` in the parsers: [sweet_xml](https://github.com/ex-aws/ex_aws_sqs/blob/master/lib/ex_aws/sqs/sweet_xml_parser.ex#L278) & [saxy](https://github.com/ex-aws/ex_aws_sqs/blob/master/lib/ex_aws/sqs/saxy_parser.ex#L349)

**Comment by TattdCodeMonkey** [comment]
The easy fix for this would be to never convert the values from a string to an integer. But this would be a breaking change since there are attributes for other responses that are integers. Like `SentTimestamp` and `ApproximateReceiveCount` for example on [ReceiveMessage](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_ReceiveMessage.html). 

A better fix would be to have some sort of schema for each message type to know what the expected types of attributes are. But I'm not sure how practical that would really be and what the level of effort for that kind of change would be.

**Comment by TattdCodeMonkey** [comment]
fix released in v3.3.1
